docs/detections/add-exceptions.asciidoc

```diff
@@ -129,22 +129,16 @@ Closes all alerts that match the exception's conditions and were generated only
 [[endpoint-rule-exceptions]]
 === Add {elastic-endpoint} exceptions
 
-Like detection rule exceptions, you can add Endpoint agent exceptions either by editing the Endpoint Security rule or by adding them as actions on alerts generated by the Endpoint Security rule. {elastic-endpoint} alerts have the following fields:
+Like detection rule exceptions, you can add {elastic-endpoint} exceptions by adding exceptions to <<endpoint-protection-rules, endpoint protection rules>>. You can also add Endpoint exceptions to rules that are associated with {elastic-endpoint} rule exceptions. To associate rules when creating or editing a rule, select the <<rule-ui-advanced-params, *{elastic-endpoint} exceptions*>> option.
```

Comment:
This phrase could be simplified: "you can add {elastic-endpoint} exceptions by adding exceptions to <<endpoint-protection-rules, endpoint protection rules>>."